### DISCUSSION TITLE
Benchmarking Minor Question

### DISCUSSION THREAD

**Discussion by Mokerak** [openzfsonwindows/openzfs#312] [General] [answered]
I'm using Crystal Disk Benchmark to try and see how much better ZFS is, but I realized most tests it does absolutely get clobbered by the fast ZFS just caches all the data its testing with. I was wondering if I could somehow test if ZFS is just inherently a better file system without using ram as a crutch vs windows Storage Spaces?

So would setting the Arc max ram to be basically nothing work to do this? Can ZFS even run without any ram? 

I guess the TL:DR is, is ZFS better at improving raw read/write speeds without caching

**Comment by Mokerak** [comment, marked as answer]
I guess after thinking about it for Gaming which I'd use, it seems like ZFS's really great feature is even if Game Engines fail to load stuff into ram that they maybe should do, ZFS can just shove stuff into ram on its own

**Comment by guenther-alka** [comment]
ZFS uses RAM as read/write cache on a read last/read most base of ZFS datablocks. Regarding reads you can disable firstlevel (or SSD secondlevel) readcaches. But be aware that performance is quite bad then. Use this only to compare disks. 

Usually do tests with cache enabled and random data. Results are then quite comparable and valid. 
If you disable caches even access to metadata is not cached so this is worse as ZFS reads metadata quite heavily.
